### PR TITLE
Feature/main

### PR DESCRIPTION
**Pull request by raxkumar-wedaa**
- limit project name size

- description size - 120, default text: No description available.

- correct project name in architectures page

- responsive tiles

- selectable texts

- remove draft counts

- remove authentication for canvas to code

- Update navbar

### CODE REVIEW

File: src/pages/Designer.js [hunk 1]
```diff
@@ -1199,17 +1196,22 @@ const Designer = ({ update, viewMode = false }) => {
...
     //
 
     const [isGenerating, setIsGenerating] = useState(false);
     const [image, setImage] = useState(null);
 
-    const generateZip = async () => {
-        const Data = generatingData;
+    const generateZip = async (e, data = null) => {
+        const Data = data || generatingData;
+        console.log(Data)
```
Comment (raxkumar-wedaa) [inline src/pages/Designer.js:1214]: remove console log

File: src/pages/Designer.js [hunk 2]
```diff
@@ -1494,12 +1496,36 @@ const Designer = ({ update, viewMode = false }) => {
     const [selectedColor, setSelectedColor] = useState('');
 
     const handleColorClick = color => {
-        let UpdatedNodes = { ...nodes };
+        console.log(color);
+        let UpdatedNodes = structuredClone(nodes);
         setSelectedColor(color);
         (UpdatedNodes[nodeClick].style ??= {}).backgroundColor = color;
-        setNodes(UpdatedNodes);
+        console.log(nodeClick, UpdatedNodes);
```
Comment (raxkumar-wedaa) [inline src/pages/Designer.js:1503]: remove console log

File: src/components/DownloadButton.js [hunk 3]
```diff
@@ -33,8 +33,8 @@ function DownloadButton() {
     };
 
     return (
-        <Button colorScheme="blackAlpha"  onClick={onClick}>
-            Download Image
+        <Button colorScheme="blackAlpha" size="sm" onClick={onClick}>
+            Export to PNG
```
Comment (rohith-wedaa) [inline src/components/DownloadButton.js:37]: Label could be - **Save as Image**

File: src/pages/Designer.js [hunk 4]
```diff
@@ -1494,12 +1496,36 @@ const Designer = ({ update, viewMode = false }) => {
     const [selectedColor, setSelectedColor] = useState('');
 
     const handleColorClick = color => {
-        let UpdatedNodes = { ...nodes };
+        console.log(color);
```
Comment (rohith-wedaa) [inline src/pages/Designer.js:1499]: remove console log

File: src/pages/Rewiew/CodeReview.js [hunk 5]
```diff
@@ -45,12 +51,34 @@ function CodeReview({ nodeId, generateMode = false, deployementData = null }) {
...
-        </Box>
+            {/* <Button
+                hidden={!generateMode || tabIndex === 3}
+                mx={4}
+                my={2}
+                colorScheme="blue"
+                onClick={tabIndex !== 3 ? () => setTabIndex(3) : () => {}}
+            >
+                {tabIndex === 3 ? 'Generate with Infrastructure' : 'Configure Infrastructure'}
+            </Button> */}
```
Comment (rohith-wedaa) [inline src/pages/Rewiew/CodeReview.js:71]: No commented code

File: src/pages/Rewiew/Infrastructure.js
Comment (rohith-wedaa) [inline src/pages/Rewiew/Infrastructure.js:1]: Since Infrastructure is moved to Review component, the original one can be removed.

**Review by rohith-wedaa — CHANGES_REQUESTED** [review]

**Review by raxkumar-wedaa — COMMENTED** [review]
Please resolve the comments if anything is pending #Dheeraj

Merging for now

**Pull request merged**